internal/helpers/httpclient.go

```diff
@@ -0,0 +1,59 @@
+package helpers
+
+import (
+	"net/http"
+	"net/url"
+	"os"
+
+	"github.com/gokrazy/internal/config"
+	"github.com/gokrazy/internal/httpclient"
+	"github.com/gokrazy/internal/updateflag"
+)
+
+func GetHTTPClient(instance string) (*http.Client, *url.URL, error) {
```

Comment:
Could this function instead live in the github.com/gokrazy/internal/httpclient package, with the rest of the shared httpclient functionality? :)

Reply:
It can, I was just being lazy and didn't wanna file two PRs :p